### PR TITLE
Duplicate All Build YAML Files 

### PR DESCRIPTION
**Pull request by rchauhan18**
Duplicate All Build YAML Files 
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/wpf/pull/8908)

### CODE REVIEW

**Review by singhashish-wpf — APPROVED** [review]

File: eng/pipeline-pr.yml [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+      pool:
+        # For public jobs, use the hosted pool.  For internal jobs use the internal pool.
+        # Will eventually change this to two BYOC pools.
+        # agent pool can't be read from a user-defined variable (Azure DevOps limitation)
+        ${{ if eq(variables['System.TeamProject'], 'public') }}:
+          name: NetCore-Public
+          demands: ImageOverride -equals windows.vs2022preview.amd64.Open
+        ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+          name: NetCore1ESPool-Internal
+          demands: ImageOverride -equals windows.vs2022preview.amd64
```
Comment (RussKie) [inline eng/pipeline-pr.yml:52]: This should be simplified - this is public-only definition now.
Ditto above

File: eng/pipeline-pr.yml [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+        - ${{ if ne(variables['System.TeamProject'], 'internal') }}:
+          - name: _InternalRuntimeDownloadArgs
+            value: ''
+        - ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+          - group: DotNetBuilds storage account read tokens
+          - group: AzureDevOps-Artifact-Feeds-Pats
+          - name: _InternalRuntimeDownloadArgs
+            value: >-
+              /p:DotNetRuntimeSourceFeed=https://dotnetbuilds.blob.core.windows.net/internal
+              /p:DotNetRuntimeSourceFeedKey=$(dotnetbuilds-internal-container-read-token-base64)
```
Comment (RussKie) [inline eng/pipeline-pr.yml:96]: This should be removed now

File: eng/pipeline-pr.yml [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+          value: ${{ parameters.repoName }}/$(Build.SourceBranch)
+        - name: _HelixToken
+          value: ''
+        - name: _HelixCreator
+          value: ${{ parameters.repoName }}
+        - name: _programfilesx86
+          value: ${Env:ProgramFiles(x86)}/dotnet          
+        - name: _programfiles
+          value:  ${Env:ProgramFiles}/dotnet
+        - ${{ if ne(variables['System.TeamProject'], 'internal') }}:
```
Comment (RussKie) [inline eng/pipeline-pr.yml:87]: This line can now be removed

File: eng/pipeline-pr.yml [hunk 4]
```diff
@@ -0,0 +1,310 @@
...
+          - name: _PublicBuildPipeline
+            value: false
+          - name: _HelixSource
+            value: official/${{ parameters.repoName }}/$(Build.SourceBranch)
+          - name: _HelixToken
+            value: '$(HelixApiAccessToken)' # from DotNet-HelixApi-Access group
+          - name: _HelixCreator
+            value: '' #if _HelixToken is set, Creator must be empty
+          - name: _TestHelixAgentPool
+            value: 'Windows.10.Amd64.ClientRS5' # Preferred: 'Windows.10.Amd64%3bWindows.7.Amd64%3bWindows.10.Amd64.ClientRS5'
```
Comment (RussKie) [inline eng/pipeline-pr.yml:124]: This should be removed now

File: eng/pipeline-pr.yml [hunk 5]
```diff
@@ -0,0 +1,310 @@
...
+
+      strategy:
+        matrix:
+          ${{ if eq(parameters.runAsPublic, 'true') }}:
```
Comment (RussKie) [inline eng/pipeline-pr.yml:128]: These conditionals are now redundant

File: eng/pipeline-pr.yml [hunk 6]
```diff
@@ -0,0 +1,310 @@
...
+        displayName: Pre-Build - Set VSO Variables
+
+      - ${{ if ne(variables['System.TeamProject'], 'public') }}:
+        - task: PowerShell@2
+          displayName: Setup Private Feeds Credentials
+          inputs:
+            filePath: $(Build.SourcesDirectory)/eng/common/SetupNugetSources.ps1
+            arguments: -ConfigFile $(Build.SourcesDirectory)/NuGet.config -Password $Env:Token
+          env:
+            Token: $(dn-bot-dnceng-artifact-feeds-rw)
```
Comment (RussKie) [inline eng/pipeline-pr.yml:168]: This should be removed now

File: eng/pipeline-pr.yml (hunk 2 above)
Comment (mmitche) [reply eng/pipeline-pr.yml:96]: There are internal PRs, not just public PRs. This should not be removed.

File: eng/pipeline-pr.yml (hunk 3 above)
Comment (mmitche) [reply eng/pipeline-pr.yml:87]: PRs can still be run internally.

Comment (RussKie) [reply eng/pipeline-pr.yml:87]: But this condition is about "project being internal" which is always true now. There's nothing about PRs here... What am I missing?

**Comment by rchauhan18** [comment]
@RussKie I have added a PR with clean-up changes for public and internal pipeline.
Here is the PR: https://github.com/dotnet/wpf/pull/8927
Please review.

**Pull request merged**